autoinit/util/visualize_init.py

```diff
@@ -87,9 +87,11 @@ def _load_models(self, model, custom_objects=None):
             raise TypeError('The model argument must be a TensorFlow Model, config dictionary, '\
                            f'or JSON string.  Got type {type(model)}.')
 
-        # Create the universal model based on the default model
+        # Create the AutoInit model based on the default model
         self.autoinit_model, self.mean_var_estimates = \
-            AutoInit().initialize_model(self.default_model.get_config(), True)
+            AutoInit().initialize_model(self.default_model.get_config(),
+                                        return_estimates=True,
+                                        custom_objects=custom_objects)
```

Comment:
Need to pass custom objects dictionary for visualization to work with custom models.